### DISCUSSION TITLE
Add option for hysteresis to ignore the set points on a graph per point

### DISCUSSION THREAD

**Discussion by BlueCrunchmaican** [Rem0o/FanControl.Releases#1690] [Ideas]
what do you think?

**Comment by Rem0o** [comment]
Not really clear by just reading your title. Try and detail the behavior you are thinking about.

**Reply by BlueCrunchmaican** [reply]
Basically each point that you set on the graph could have its own properties for whether or not it's ignored by hysteresis. A checkbox that would enable/disable it obviously but I would like to be able to have certain spd/tmp point able to ignore hysteresis while others do not.

**Reply by Rem0o** [reply]
And what's the use case? And what if you cross that point, hysteresis is always ignored, both ways? 

Let say you ignore at 50, and you go from 40 to 60. The problem is that the program never really is on a point, it's somewhere on the curve. There's a linear interpolation between points. You take the closest point? 

You can already mix your graph with a trigger, that might be the behaviour you are after.
